### PR TITLE
Added frame rate limiting code to PySDL2 example for issue #324

### PR DESCRIPTION
**Pull request by neilmunday**
Added frame rate limiting code to PySDL2 example for issue #324.

I have left in debug code that will print the frame rate if uncommented.

Perhaps we could use optparse to add debug flags to the example?

Note: only tested under Linux

### CODE REVIEW

File: examples/pysdl2.py [hunk 1]
```diff
@@ -36,33 +43,43 @@
   https://github.com/neilmunday/pes/blob/master/lib/pes/ui.py
 """
 
+import argparse
+import logging
 import sys
+
```
Comment (cztomczak) [inline examples/pysdl2.py]: PEP 8 requires two blank lines here. There are several PEP8 issues in this commit that should be fixed. There is a tool that can check code for pep8 issues.

File: examples/pysdl2.py [hunk 2]
```diff
@@ -269,6 +297,18 @@ def main():
             sdl2.SDL_Rect(0, headerHeight, browserWidth, browserHeight)
         )
         sdl2.SDL_RenderPresent(renderer)
+
+        # FPS debug code
+        if logLevel == logging.DEBUG:
+            frames += 1
```
Comment (cztomczak) [inline examples/pysdl2.py]: My editor PyCharm displays two issues with "frames" and "lastFrameTick" variables ("local variable might be referenced before assignment"). Please move declaration of variables before the "if" block codes to get rid of these notices.

PyCharm editor has a free version and you can download cefpython configuratoin file for PyCharm in Issue #232.

**Comment by cztomczak** [comment]
After your previous PR I've made multiple fixes to the code to get rid of PEP8 issues and other issues detected by PyCharm. All examples must conform to PEP8 and pycharm code style, so that I can see a green tick in PyCharm and detect issues/bugs in scripts early.

**Comment by neilmunday** [comment]
I've made the requested changes.

I'm usting Atom (now with pep8 plugins) btw.

**Comment by cztomczak** [comment]
Thank you.

**Pull request merged**